spec/mongoid/document_fields_spec.rb

```diff
@@ -85,4 +85,19 @@
       end
     end
   end
+
+  describe 'Regexp field' do
+    class Template
```

Comment:
It is my understanding that class definitions in Ruby are global (even though this one appears to be scoped to the rspec group). Therefore I suggest one of the following instead:

- Renaming the class to have the scope in its name, e.g. DocumentFieldsSpecTemplate.
- Moving it to document_fields_models.rb file in the current directory, with a namespaced name like DocumentFieldsModels::Template or DfmTemplate.
- Putting the class into our (huge) models directory.
- Reusing an existing class from the models directory without modifications to that class (additions can be OK depending on what they are).

Reply:
Good catch, thanks! I used one of existing classes from models.